SCClassLibrary/Common/Core/Kernel.sc

```diff
@@ -452,11 +462,28 @@ FunctionDef {
 		_FunctionDefDumpContexts
 		^this.primitiveFailed
 	}
+
+	// These methods are used to indicate that only a subsection of the argName
+	//     and prototypeFrame are used to call the FunctionDef.
```

Comment:
Would it be OK to go back a little bit and minimise the footprint? The commenting of the variable names is OK, but the rest feels a little bit too extensive for the current PR. I'll propose some "reductions", sorry – I know this is a lot of work.